projects/core/components/primitive-textfield/value-decoration/value-decoration.style.less

```diff
@@ -19,6 +19,23 @@
     text-overflow: clip;
 }
 
+.t-postfix {
+    display: inline-block;
+    line-height: 0;
+    margin-top: 1.15ch;
+
+    &:not(&--is-empty):before {
+        /**
+         @note(splincode): emulate native whitespace before input value
+         */
+        content: '\00a0';
```

Comment:
Currently whitespace is added through the getter so this makes two whitespaces. And the reason it is added with a getter is `%` sign has to be shown without space according to our typographic guidelines :) If you really want to move it to CSS it can be done though